static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -1022,17 +1042,24 @@ const Body = styled(Layout.Body)`
 `;
 
 const Main = styled(Layout.Main)`
+  max-width: 1000px;
   padding: ${space(4)};
+  flex: 1;
```

Comment:
I believe some code I added to the Body was also removed, regressing the UI

![image](https://user-images.githubusercontent.com/29228205/156726177-ba8e3d84-959d-4666-b045-b3b27f1a036c.png)

Reply:
I added them back in this PR https://github.com/getsentry/sentry/pull/32309 ^^